app/controllers/terms_and_conditions_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Controller that manages terms and conditions
+class TermsAndConditionsController < ApiController
+  def index
+    render json: TermsAndCondition.find(1).slice(:link)
```

Comment:
Why is the index hardcoded to `find(1)`?

Reply:
So we want there to only be one of the Terms and Conditions. I didn't know how to do that limitation any other way :sweat_smile:

Reply:
Then why not use `show` or some custom naming? I think it'd be more explicit given that you'll only receive one object, and not "all".

Reply:
Okay, I'll change it to show :+1:

Reply:
Done!!